src/utils.ts

```diff
@@ -339,6 +344,16 @@ export class Utils {
             process.env.GITHUB_SERVER_URL + '/' + process.env.GITHUB_REPOSITORY + '/actions/runs/' + process.env.GITHUB_RUN_ID,
         );
         Utils.exportVariableIfNotSet('JFROG_CLI_USER_AGENT', Utils.USER_AGENT);
+
+        // Set JF_PROJECT as JF_CLI_BUILD_PROJECT to allow the JFrog CLI to use it as the project key
+        let projectKey: string | undefined = process.env.JF_PROJECT;
```

Comment:
This addition should be documented in the README

Reply:
https://github.com/jfrog/setup-jfrog-cli/pull/154